cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1296,10 +1296,10 @@ instance Show exception => ToObject (TraceLocalRootPeers RemoteAddress exception
              , "domainAddress" .= toJSON d
              , "diffTime" .= show dt
              ]
-  toObject _verb (TraceLocalRootResult d res) =
+  toObject _verb (TraceLocalRootResult d _) =
     mkObject [ "kind" .= String "LocalRootResult"
              , "domainAddress" .= toJSON d
-             , "result" .= Aeson.toJSONList res
+             , "result" .= String "Unk"
```

Comment:
Why aren't we using `res` anymore?

Reply:
This was from a leftover that was updated thanks to cherry-picking other commits from the Network team